ggshield/core/scan/file.py

```diff
@@ -61,6 +61,7 @@ def get_files_from_paths(
     display_scanned_files: bool,
     display_binary_files: bool,
     ignore_git: bool = False,
+    ignore_git_staged: bool = False,
```

Comment:
If I understand correctly, the `get_files_from_paths` function was previously always taking staged files.
If we introduce a boolean to toggle this, I think its default value should be what the function was previously doing. Especially since this function is used in 5 different files including Secret and SCA files. Are we sure we want to change the default behavior for them as well ?

Reply:
The default value does correspond to the previous behaviour.